core/src/main/java/org/elasticsearch/search/aggregations/support/ValuesSourceParser.java

```diff
@@ -66,7 +70,7 @@ public static Builder any(String aggName, InternalAggregation.Type aggType, Sear
         return new Builder<>(aggName, aggType, context, ValuesSource.GeoPoint.class).targetValueType(ValueType.GEOPOINT).scriptable(false);
     }
 
-    private static class Input {
+    public static class Input {
         String field = null;
```

Comment:
Lookig at this again, I don't like exposing the Input just to get the timezone information in the DateHistogramParser, I think it better belongs there where it was in the first place. Will add a commit where I change to passing timezone as optional argument to `ValueSourceParse.config()`, IMHO much cleaner.